src/chrome/content/rules/Open_Source.org.xml

```diff
@@ -5,6 +5,7 @@
 <ruleset name="Open Source.org (partial)">
 
 	<target host="opensource.org" />
+	<target host="www.opensource.org" />
 
```

Comment:
Would you please 

- [ ] Add `lists.opensource.org`
- [ ] Remove the obsolete top comment
- [ ] Delete the ruleset file `Open_Source.org-problematic.xml`

Reply:
Done.